app/src/main/res/values/colors.xml

```diff
@@ -14,6 +14,7 @@
   <color name="oppiaPrimaryText30">#4D333333</color>
   <color name="oppiaPrimaryDark">#00645C</color>
   <color name="oppiaSecondaryText">#666666</color>
+  <color name="oppiaDarkGreen">#008098</color>
```

Comment:
Remove this color. It is not getting used anywhere.

Reply:
Removed.